src/main/kotlin/com/autonomousapps/model/Source.kt

```diff
@@ -82,8 +82,8 @@ data class AndroidResSource(
   data class AttrRef(val type: String, val id: String) {
     companion object {
 
-      private val TYPE_REGEX = Regex("""@(?:.+:)?(.+)/""")
-      private val ATTR_REGEX = Regex("""\?(?:.+/)?(.+)""")
+      private val TYPE_REGEX = Regex("""@(\+|\w+:)?(?<type>\w+)/(\w+)""")
+      private val ATTR_REGEX = Regex("""\?((\w+:\w+|\w+)/)?(?<attr>\w+)""")
```

Comment:
I'm going to need to spend a few minutes studying these regexes to ensure I understand them 😅

Reply:
:D if that helps in any way, I can add dedicated unit tests for that as well.

Reply:
And here is an easier way to "visualize" them with examples:
- https://regex101.com/r/N3ZPTB/1
- https://regex101.com/r/GH2uoF/1

Reply:
If you want you could add tests, but even just a comment explaining what each component of the regex does would be _super_ helpful.

Reply:
👍 I've been able to find links to the official documentation regarding these regexes.

Reply:
`?android:foo` seems to be valid as well, I'll update the regex accordingly